class/class3.2/src/Helpers.cpp

```diff
@@ -0,0 +1,24 @@
+#include "../include/Helpers.hpp"
+
+double Triangle::calculateArea() {
+    double s = (a + b + c) / 2;
```

Comment:
some validation of a, b, c is needed as non-positive values or values that violate  inequality of triangle will cause an issue iinside formula.

Reply:
and return type std::optional<double> will be more relevant here